ruby/rails/security/injection/rails-check-json-parsing-rce.yaml

```diff
@@ -0,0 +1,67 @@
+rules:
```

Comment:
this one is only for CVE-2013-0333 and not for CVE-2013-0269 right?

Reply:
Yes, I'll be working on one for 0269 next when I have time to do more Brakeman rules!